lib/pagseguro/transaction/serializer.rb

```diff
@@ -32,13 +32,13 @@ def serialize_status_history
 
       private
       def serialize_general(data)
-        data[:code] = xml.css("transaction > code").text
-        data[:reference] = xml.css("transaction > reference").text
-        data[:type_id] = xml.css("transaction > type").text
-        data[:payment_link] = xml.css("transaction > paymentLink").text
-        data[:status] = xml.css("transaction > status").text
+        data[:code] = xml.css("code")[0].text
```

Comment:
But I don't understand this yet, because if you call the text method the result will be the same for you just have 1 code tag...
```
xml.css("code").text == xml.css("code")[0].text
```
Just in cases where you find more than 1 code tag you should use [0] for take the first.